packages/imagemin/README.md

```diff
@@ -0,0 +1,125 @@
+# Neutrino Imagemin Middleware
+
+`@neutrinojs/imagemin` is Neutrino middleware for optimizing images with imagemin.
+
+[![NPM version][npm-image]][npm-url]
+[![NPM downloads][npm-downloads]][npm-url]
+[![Join the Neutrino community on Spectrum][spectrum-image]][spectrum-url]
+
+## Requirements
+
+- Node.js v6.10+
+- Yarn or npm client
+- Neutrino v7
+
+## Installation
+
+`@neutrinojs/imagemin` can be installed via the Yarn or npm clients.
+
+#### Yarn
+
+```bash
+❯ yarn add @neutrinojs/imagemin
+```
+
+#### npm
+
+```bash
+❯ npm install --save @neutrinojs/imagemin
+```
+
+## Usage
+
+`@neutrinojs/imagemin` can be consumed from the Neutrino API, middleware, or presets. Require this package
+and plug it into Neutrino:
+
+```js
+// Using function middleware format
+const images = require('@neutrinojs/image-loader');
+const imagemin = require('@neutrinojs/imagemin');
+
+// Use with default options
+neutrino.use(images);
+neutrino.use(imagemin);
+
+// Usage showing default options
+neutrino.use(imagemin, {
+	imagemin: {
+	  plugins: [
+	    gifsicle(),
+	    svgo(),
+	    pngquant(),
+	    mozjpeg(),
+	    webp()
+	  ]
+	},
+	plugin: {
+	  name: '[path][name].[ext]',
+	  test: /\.(png|jpg|jpeg|gif|webp)$/
+	},
+	pluginId: 'imagemin',
+	useId: 'imagemin',
+	rules: ['svg', 'img']
+});
+```
+
+```js
+// Using object or array middleware format
+
+// Use with default options
+module.exports = {
+  use: ['@neutrinojs/imagemin']
+};
+
+// Usage showing default options
+module.exports = {
+  use: [
+    ['@neutrinojs/imagemin', {
+			imagemin: {},
```

Comment:
Heh, looks like we may have some mixed tabs in here. Make each indentation here 2 spaces.

Reply:
Oh dear…sorry in the middle of changing-editors-hell 🥇